OpenSim/Simulation/Model/Model.cpp

```diff
@@ -2033,9 +2020,9 @@ void Model::formQStorage(const Storage& originalStorage, Storage& qStorage) {
     Array<string> qNames;
     getCoordinateSet().getNames(qNames);
 
+    std::vector<int> mapColumns(qNames.getSize());
```

Comment:
This version does not fill the tail with `-1` default vals, so need to ensure that doesn't break something